src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_map.scss

```diff
@@ -11,17 +11,7 @@ category: Widgets
         @include clearfix;
     }
 
-    .map-buttons .map-button-save {
-        &, &:hover, &:focus {
-            color: $color-brand-two-introvert;
-        }
-
-        &:active {
-            color: $color-brand-one-introvert;
-        }
-    }
-
-    .map-button-save, .map-button-cancel {
+    .map-button-cancel {
```

Comment:
You are right. I thought it should be used for the two new buttons. Could you please fix this?